[docs] observe docs updates (#46370)
expo/expo: docs/pages/eas/observe/eas-update.mdx

---
title: EAS Update download performance
sidebar_title: EAS Update
description: Track how long EAS Update OTA bundles take to download on real user devices, with per-update breakdowns in the EAS Observe dashboard.
---

import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
import { Terminal } from '~/ui/components/Snippet';

> EAS Observe automatically tracks how long each OTA update takes to download on real user devices. No additional instrumentation is required: if your app uses [EAS Update](/eas-update/introduction/) and includes [`expo-observe`](/eas/observe/get-started/), download metrics are collected for every update fetch.

Reviewer: **[suggestion]** The second sentence stacks two passive clauses ("is required", "are collected"). Active voice addresses the reader directly and shortens the sentence (`expo-docs-style-guide.md#voice-and-tone`).

```suggestion
EAS Observe automatically tracks how long each OTA update takes to download on real user devices. You don't need to add any instrumentation: if your app uses [EAS Update](/eas-update/introduction/) and includes [`expo-observe`](/eas/observe/get-started/), EAS Observe collects download metrics for every update fetch.
```

Update download times appear in the **EAS Update** tab of the EAS Observe dashboard and are queryable from the EAS CLI.

## View update downloads

> In the dashboard: open your project and navigate to [**Observe > EAS Update**](https://expo.dev/accounts/[account]/projects/[project]/observe?tab=eas-update). The tab is split into two sections: an aggregate chart and a per-update table.

Reviewer: **[suggestion]** "is split" is passive. "Has" reads more direct (`expo-docs-style-guide.md#voice-and-tone`).

```suggestion
In the dashboard: open your project and navigate to [**Observe > EAS Update**](https://expo.dev/accounts/[account]/projects/[project]/observe?tab=eas-update). The tab has two sections: an aggregate chart and a per-update table.
```

<ContentSpotlight
  alt="The EAS Update tab in the EAS Observe dashboard showing update download time and recent updates."
  src="/static/images/expo-observe/observe-updates-light.png"
  darkSrc="/static/images/expo-observe/observe-updates-dark.png"
/>

### Update download time

A single chart showing aggregate download time across all updates fetched in the selected time range. Statistical breakdowns mirror the App startup tab: **Median**, **Avg**, **Min**, **Max**, **P90**, and **P99**. Use these to spot regressions in update size or CDN latency.

> Update markers on the chart indicate when each update was first downloaded. Click a marker to see the update ID, version, and metrics at that point.

Reviewer: **[suggestion]** "was first downloaded" is passive. Active reads tighter (`expo-docs-style-guide.md#voice-and-tone`).

```suggestion
Update markers on the chart indicate when each update first downloaded. Click a marker to see the update ID, version, and metrics at that point.
```
